components/raftstore/src/store/fsm/apply.rs

```diff
@@ -166,21 +166,42 @@ where
     }
 }
 
-#[derive(Default, Debug)]
+#[derive(Default)]
 pub struct ChangePeer {
     pub index: u64,
     pub conf_change: ConfChange,
     pub peer: PeerMeta,
     pub region: Region,
 }
 
-#[derive(Debug)]
+impl Debug for ChangePeer {
```

Comment:
Why not just `derive(Debug)`?

Reply:
I forget that we've changed all `PbPrint` structures, I'll change it.